src/services/models/iotHubManagerModels.js

```diff
@@ -167,3 +167,7 @@ export const toNewDeviceRequestModel = ({
         }
   };
 }
+
+export const toDevicePropertiesModel = (iotResponse, dsResponse) => {
+  return [...(new Set([...getItems(iotResponse), ...getItems(dsResponse)]))];
```

Comment:
This will work but as mentioned it would probably be helpful to break this down into an extra variable or two for readability purposes. Maybe break the set into a separate variable before converting it into an array.